mls-rs-crypto-webcrypto/src/lib.rs

```diff
@@ -0,0 +1,333 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright by contributors to this project.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#[cfg(mls_build_async)]
```

Comment:
Since every definition seems to be conditionally compiled, would it be simpler to conditionally compile the entire file? This way all `#[cfg(mls_build_async)]` can be removed and a single `#![cfg(mls_build_async)]` can be added.

Reply:
Thank you for the suggestion! Added now.